pkg/cluster/nodes/node.go

```diff
@@ -71,6 +71,7 @@ func (n *Node) Command(command string, args ...string) exec.Cmd {
 type nodeCache struct {
 	kubernetesVersion string
 	ip                string
+	ipv6              string
```

Comment:
so https://golang.org/pkg/net/#IP is technically already an abstraction between v4 and v6 can we use that?

Reply:
I was thinking that if we'll need to support dual stack it can be useful to have both fields.
Also, I think that will imply more changes in other parts of the code that are not relevant to this PR